Added guidelines for code formatting on save (#175)
EtherealEngine/etherealengine-docs: docs/manual/98_contributing/05_source/60_codeFormatting.md

# Code Formatter

We use prettier to format our code.

## Format on Save

### Setting Up Prettier in VS Code

1. **Install Prettier Extension (if not already installed):**
   - Press `Ctrl` + `P` to open the command palette.
   - Type `ext install esbenp.prettier-vscode` and press `Enter`.
   - Follow the prompts to install the Prettier extension.

2. **Open Settings:**
   - Press `Ctrl` + `,` together to open the VS Code settings.

> 3. **Enable Format on Save:**
>    - In the search bar at the top, type `Format on Save`.
>    - You will see three tabs: User, Remote, and Workspace.
>    - Check the box next to `Editor: Format on Save` under both User and Remote tabs. **Do not check the box under the Workspace tab.**

Reviewer: ```suggestion
### Step 3. Enable Format on Save
   1. In the search bar at the top, type `Format on Save`.
   2. You will see three tabs: **User**, **Remote**, and **Workspace**.
   3. Check the box next to **Editor: Format on Save** under the User and Remote tabs. 
   
   :::note

   Do not check the box under the **Workspace** tab.

   :::
```

> 4. **Set Prettier as Default Formatter:**
>    - Open any file in your workspace.
>    - Right-click within the file editor.
>    - Select `Format Document With...`.
>    - Choose `Configure Default Formatter...`.
>    - Select `Prettier - Code Formatter` from the list.

Reviewer: ```suggestion
### Step 4. Set Prettier as the Default Formatter
   1. Open any file in your workspace.
   2. Right-click within the file editor.
   3. Select **Format Document With...**.
   4. Choose **Configure Default Formatter...**.
   5. Select **Prettier - Code Formatter** from the list.
```

> ### Verifying Your Setup

Reviewer: ```suggestion
## Verifying Your Setup
```

Moved this header one level up to fit the proposed structure.

> 1. **Test Format on Save:**
>    - Open a file with some unformatted code.
>    - Make a small edit and save the file (`Ctrl` + `S`).
>    - Verify that the code is automatically formatted upon saving.

Reviewer: I removed the single-item bullet point and improved the content for readability.

```suggestion
To ensure code is automatically formatted upon saving:

1. Open a file with some unformatted code.
2. Make a small edit and save the file (`Ctrl` + `S`).
3. Check if the code is automatically formatted upon saving.
```

# Code Formatting

Prettier helps us maintain consistent code formatting across our projects.

## Setting up Prettier in VS Code

### Step 1. Install the Prettier Extension
   1. Press `Ctrl` + `P` to open the command palette.
   2. Type `ext install esbenp.prettier-vscode` and press `Enter`.
   3. Follow the prompts to install the Prettier extension.

> ### Step 2. **Open VS Code Settings**

Reviewer: ```suggestion
### Step 2. Open VS Code Settings
```
